### DISCUSSION TITLE
Probably nothing is working...

### DISCUSSION THREAD

**Discussion by Gyurmatag** [nuxt/nuxt#17595] [Questions]
Hex, just installed the fresh rc.3 and probably nothing is working, I cannot push buttons, cannot submit forms with vee-validate, nothing. When using the rc.0 it was working, now everything is a mess. Here is my project info:

- Operating System: `Darwin`
- Node Version:     `v16.14.0`
- Nuxt Version:     `3.0.0-rc.3`
- Package Manager:  `npm@8.3.1`
- Builder:          `vite`
- User Config:      `buildModules`, `modules`, `intlify`, `alias`, `css`, `colorMode`, `build`, `publicRuntimeConfig`, `meta`
- Runtime Modules:  `@nuxtjs/color-mode@3.0.3`
- Build Modules:    `@intlify/nuxt3@0.1.10`, `@pinia/nuxt@0.1.9`

Anyone experienced this?

**Comment by danielroe** [comment]
Would you provide a reproduction? 🙏

**Comment by joshuar500** [comment]
Hi guys, I'm experiencing the same thing. Just upgraded to rc3. I am using FormKit for forms with the @submit prop. When I click the submit button it never calls the function. Instead just adds the form fields as a query param to the url.

I am having issues trying to create something reproducible, but the code and setup looks like this:
https://codesandbox.io/s/thirsty-breeze-lncy9f?file=/app.vue

Downgrading to rc.2 fixes the issue for me.

~EDIT: not sure if it's related but I'm getting:~
~`Uncaught SyntaxError: import not found: isEqual` at `router.mjs` on load~
I see this has been answered here: https://github.com/nuxt/framework/issues/4883
